### PR TITLE
Add Rockpool

### PR DESCRIPTION
**Pull request by ai-cortex**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by DylanMuir** [comment]
I am willing to be listed as package maintainer.

**Comment by ai-cortex** [comment]
Dear @conda-forge/help-python, this pull request is now ready for review.
Thanks, Dylan @DylanMuir

File: recipes/rockpool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3ded42eba5e25abfda55f839ffe983010f423806dbfca961cba2321c00dc6c8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/rockpool/meta.yaml]: Remove empty section
```suggestion
```

File: recipes/rockpool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - python >=3.6
+    - pip
+  run:
+    - numpy
+    - scipy
+    - numba
+    - python >=3.6
+
+test:
+  requires:
```
Comment (xhochy) [inline recipes/rockpool/meta.yaml]: These shouldn't be required for the import test

**Comment by ai-cortex** [comment]
@xhochy Thanks for your suggestions. I've made the corrections you advised.

File: recipes/rockpool/meta.yaml (hunk 1 above)
Comment (ai-cortex) [reply recipes/rockpool/meta.yaml]: Removed

File: recipes/rockpool/meta.yaml (hunk 2 above)
Comment (ai-cortex) [reply recipes/rockpool/meta.yaml]: Removed

File: recipes/rockpool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - numba
+    - python >=3.6
+
+test:
+  imports:
+    - rockpool
+
+about:
+  home: https://gitlab.com/ai-ctx/rockpool
+  license: AGPL-3.0
```
Comment (xhochy) [inline recipes/rockpool/meta.yaml]: Please add a license family.
```suggestion
  license: AGPL-3.0
  license_family: AGPL
```

**Comment by DylanMuir** [comment]
@xhochy "Test" is failing on win64, but it's unclear to me exactly why. It succeeds on linux and osx, and only raises some warnings about optional packages not being available (which shouldn't cause failure). However, `pkg_resources` seems to also be missing, which should be available with `setuptools`, shouldn't it? Could you please take a look and let me know what you think?

**Comment by xhochy** [comment]
The error looks weird, I have no idea how this is produced, sorry.

**Comment by DylanMuir** [comment]
Dear @conda-forge/help-python, could someone please assist with explaining why staged-recipes fails for win64, and complains about a missing `pkg_resources`?

**Pull request merged**